### PR TITLE
Release Notes 4/05/2021-4/22/2021

### PR DESCRIPTION
**Pull request by miwithro**

### CODE REVIEW

**Review by qpetraroia — APPROVED** [review]

**Comment by miwithro** [comment]
@keikhara can you review and approve.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
...
+* CSI Drivers will become default for [Kubernetes versions 1.21+](https://docs.microsoft.com/azure/aks/csi-storage-drivers).
+* Previous [pod security policy (preview)](https://docs.microsoft.com/azure/aks/use-pod-security-policies) deprecation was June 30th 2021. To better align with Kubernetes Upstream pod security policy (preview) deprecation will begin with Kubernetes version 1.21, with its removal in version 1.25. As Kubernetes Upstream approaches that milestone, the Kubernetes community will be working to document viable alternatives.
+* For all AKS clusters using Kubernetes v1.20+, CoreDNS will be upgraded to version 1.8.3. This will remove `resyncperiod` and `upstream`from the Kubernetes plugin.
+
+### Release Notes
+
+* New Features
+  * You can now update Windows passwords via Azure cli.
+* Bug Fixes
+  * Fixed a bug with Cert Rotation trying to call windows agentpools, this is a linux only function.
```
Comment (weinong) [inline CHANGELOG.md:19]: > this is a linux only function

this is confusing. does it mean cert rotation doesn't apply to Windows? is it documented somewhere?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+  * Fixed a bug that if a customer uses "[]" as "AvailabilityZones" for both create and update, their update will be blocked incorrectly.
+* Behavioral Changes
+  * Node pool limit has increased from 10 to 100.
+* Component Updates
+  * Linux Pause container image has been updated to [3.5] from 1.3.1
```
Comment (weinong) [inline CHANGELOG.md:24]: > [3.5] 

what's the deal with brackets?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,35 @@
...
+  * Dns-autoscaler image has been updated to [mcr.microsoft.com/oss/kubernetes/autoscaler/cluster-proportional-autoscaler:1.8.3]  for 1.18 and above cluster.  1.8.3 uses non-root user.
+  * Pod Identity nmi image has been updated to [1.7.5] and set critical addon torelations.
+  * OSM add on has been updated to [v0.8.3]
+  * The Envoy image has been updated to [1.17.2]
```
Comment (weinong) [inline CHANGELOG.md:28]: Envoy image of what feature? OSM? should we just point to OSM release note?

File: CHANGELOG.md (hunk 1 above)
Comment (miwithro) [reply CHANGELOG.md:19]: Yes cert rotation does not apply to windows.  I will updating the doc to reflect that.

**Review by weinong — APPROVED** [review]

**Pull request merged**